### PR TITLE
Continuación a la historia del 'artefacto'

### PR DESCRIPTION
**Pull request by minimakina**

### CODE REVIEW

**Comment by minimakina** [comment]
Please review this, @casey-collab

File: espanol/artefacto/resultado_explosion/alienigenas.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Estos dos seres, con un brillo multicolor pero casi transparentes,
```
Comment (casey-collab) [inline espanol/artefacto/resultado_explosion/alienigenas.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: espanol/artefacto/resultado_explosion/perderoido.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Pero obtuvieron una potenciación de la capacidad visual, pudiendo ver ahora realidades existentes
```
Comment (casey-collab) [inline espanol/artefacto/resultado_explosion/perderoido.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by minimakina** [comment]
Please review this, @casey-collab v2

**Comment by casey-collab** [comment]
Looks great! :crown:

**Pull request merged**